assets/app/view/log.rb

```diff
@@ -50,7 +50,11 @@ def render
 
       lines = @log.map do |line|
         if line.is_a?(String)
-          h(:div, line)
+          if line.start_with?('--')
```

Comment:
i think the green might be too much

also i think this may be a bit cleaner if you only do the style programatically


so
line_props = { style: {}}
line_props[:style][:fontWeight] = 'bold' if line.start_with?('--')
h(:div, line_props, line)

Reply:
Green doesn't look important. Red might...
I agree with Toby that bold might be enough to start with.

Reply:
Changed to a CSS class.

Reply:
sorry to be picky, but could we do this inline and not use css classes. 

i try to avoid css classes as much as possible.

by changing this to a CSS class, the notifications emails will not have any styling since we can't send CSS to mail providers. inline styles are more universal and easier to maintain in this project

Reply:
Oh I understand. No problem at all, thanks for explaining.

Reply:
Ok, changed back to inline styles and cleaned it up a bit. Didn't touch the user-generated messages render.

Thanks for the patience! Really appreciating the site and wanted to start trying to contribute if possible.